common/src/main/java/com/microsoft/hydralab/performance/entity/WindowsBatteryParsedData.java

```diff
@@ -60,6 +61,16 @@ public void accumulate(WindowsBatteryMetrics metrics)
                 this.timeStamp = metrics.timeStamp;
             }
         }
+
+        @Override
+        public LinkedHashMap<String, Double> getBaselineMetricsKeyValue() {
+            return null;
+        }
+
+        @Override
+        public SummaryType getSummaryType() {
+            return null;
+        }
     }
```

Comment:
@AC-Phoenix please override these methods for Windows battery and memory entity at your convenience.